src/main/java/org/mifos/connector/channel/zeebe/ZeebeWorkers.java

```diff
@@ -87,7 +93,35 @@ private void workerSendErrorToChannel() {
             client.newCompleteCommand(job.getKey()).variables(existingVariables).send();
         }).name("send-error-to-channel").maxJobsActive(workerMaxJobs).open();
     }
-
+    private void callTransferRoute(){
+        zeebeClient.newWorker().jobType("call-transfer-API").handler((client, job) -> {
```

Comment:
follow naming conventions of worker node

Reply:
i am following the naming conventions used in other workers in this file ex -  send-success-to-channel